runtime/doc/lsp.txt

```diff
@@ -25,24 +25,37 @@ Nvim provides an LSP client, but the servers are provided by third parties.
 Follow these steps to get LSP features:
 
 1. Install language servers using your package manager or by following the
-   upstream installation instruction. You can find language servers here:
+   upstream installation instructions. You can find language servers here:
    https://microsoft.github.io/language-server-protocol/implementors/servers/
 
 2. Configure the LSP client per language server. See |vim.lsp.start()| or use
    this minimal example as a guide: >lua
 
-    vim.lsp.start({
-      name = 'my-server-name',
-      cmd = {'name-of-language-server-executable'},
-      root_dir = vim.fs.root(0, {'setup.py', 'pyproject.toml'}),
+    -- Create an event handler for the FileType autocommand
```

Comment:
Nitpick: the "event handler" _is_ the "autocommand" (in Vimspeak) for the `FileType` "event".

Of course, the question is how the audience is here -- whether we should "push" Vimspeak or generic protocol language.

Reply:
Trying to avoid Vim specific jargon here. We can leave lots of breadcrumbs (tags) to other help docs to guide the user to learn more, but at this point I want to assume as little a priori knowledge as possible.